### DISCUSSION TITLE
Platonic Partners Submod Development Thread

### DISCUSSION THREAD

**Discussion by ashyashton** [Monika-After-Story/MonikaModDev#7817] [general - submod / external content]
I was told to start an issue to track the development of this project, so here I am! I am planning to develop a submod for those of us who truly do love Monika and want to spend the rest of our lives with her, just... Not in a romantic way. The goal is to make things a bit more comfortable for players who don't want things to lean into the romantic side of things, as well as introducing new, platonic partnership specific content.

I was told to include links to where I plan to develop the mod and such, so here are the links:
[Reddit thread where things started](https://www.reddit.com/r/MASFandom/comments/it9ira/who_wants_to_team_up/)
[Discord for development of the project](https://discord.gg/Pp4xJR9)
[GitHub repository for the project](https://github.com/ashyashton/MAS-Platonic-Partners-Submod)

**Comment by xXMidnight** [comment]
platonicity- that is not a word, apparently-

**Comment by ThePotatoGuy** [comment]
linking to #4849
